.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,11 +32,22 @@ Fixes #
 **Special notes for your reviewer**:
 
 **Does this PR introduce a user-facing change?**:
+
 <!--
-If no, just write "NONE" in the release-note block below.
-If yes, a release note is required:
-Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
+    Don't forget about CHANGELOG!
+
+    Changelog entry format:
+    - <PR-title>. ([#<PR-id>](<PR-URL>), [@<author-name>](<authror-URL>), ...)
+
+    <PR-title> Title of your pull request.
+    <PR-id> Id of your pull request.
+    <PR-URL> URL of your PR such as https://github.com/thanos-io/thanos/pull/<PR-id>
+    <author-name> Name of your github account.
+    <author-URL> URL of your github account.
+
+    An example:
+    - Implment CloudCore HA in hot standby mode. ([#1600](https://github.com/kubeedge/kubeedge/pull/1600), [@GsssC](https://github.com/GsssC), [@kevin-wangzefeng](https://github.com/kevin-wangzefeng)).
 -->
-```release-note
 
-```
+* [ ] I added CHANGELOG entry for this change.
+* [ ] Change is not relevant to the end user.
```

Comment:
We can revert the change for the PR template, which need more discussion

Reply:
Get it.